src/services/ieq/ieq.js

```diff
@@ -284,7 +284,11 @@ IEQ.prototype._updateData = function(callback, dataConfig, funcName){
 		}
 	}, function(dnId, err, data){
 		if(err) {
-			Logger.error("["+that.dataConfig.sensors+"] Recv err: "+JSON.stringify(err));
+			if (that.dataConfig && that.dataConfig.sensors) Logger.error("["+that.dataConfig.sensors+"] Recv err: "+JSON.stringify(err));
+			if (typeof err == "object" && typeof err.name =='string' && typeof err.error =='string') {
+				callback(null, err.name);
```

Comment:
I think the field 'error' should be renamed 'message'.  Also, I dont think it should be necessary to answer the callback.